src/test/java/com/unbappd/enterprise/EnterpriseApplicationTests.java

```diff
@@ -1,13 +1,62 @@
 package com.unbappd.enterprise;
 
+import com.unbappd.enterprise.dto.Review;
+import com.unbappd.enterprise.service.IReviewService;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 @SpringBootTest
 class EnterpriseApplicationTests {
 
+    @Autowired
+    IReviewService reviewService;
+    private Review review;
+
     @Test
     void contextLoads() {
     }
 
+    /**
+     * Validate that the DTO properties can be set and retrieved.
+     */
+    @Test
+    void verifyReviewProperties() {
+        String reviewTxt = "Really Good Beer";
+        Integer reviewScr = 5;
+
+        Review reviewEntry = new Review();
+        reviewEntry.setReviewId(0);
+        reviewEntry.setPlacesId(0);
+        reviewEntry.setReviewScore(5);
+        reviewEntry.setLongitude(51.1232);
+        reviewEntry.setLongitude(51.1232);
+        reviewEntry.setReviewText(reviewTxt);
+        reviewEntry.setReviewerID(101);
+
+        assertEquals(reviewTxt, reviewEntry.getReviewText());
+        assertEquals(reviewScr, reviewEntry.getReviewScore());
+    }
+
+    private void givenReviewDataAreAvailable() {
+    }
+    private void whenUserSearchesWithPlaceID9(){
+        review = reviewService.fetchById(9);
+
+    }
+    private void thenReturnReviewsWithPlaceID9(){
+        String returnedReviewText = review.getReviewText();
+        int checkPlacesID = review.getPlacesId();
+        assertEquals("A fabulous beer", returnedReviewText);
+        assertEquals(9,checkPlacesID);
+    }
+
+    @Test
+    void saveNewReview_validateNewReviewByReturningReviewText() {
+        givenReviewDataAreAvailable();
+        whenUserSearchesWithPlaceID9();
+        thenReturnReviewsWithPlaceID9();
+    }
```

Comment:
Use more unit tests! Look at your GIVEN WHEN THEN in your readme and make tests out of them. Here, I am using placeID to return review text using your service stub. Since your app is centered around showing users drink reviews "nearby", I assumed placeID was a nearby indicator, almost like a zipcode. "To retrieve nearby drink reviews, use placeID 9 to get reviews of that same placeID" is basically what I was testing here.